api/pkg/template/config.go

```diff
@@ -277,19 +279,11 @@ func (e *Engine) configValueChanged(itemName string) bool {
 		return true
 	}
 
-	return prevVal.Value != currentVal.Value
-}
-
-func (e *Engine) getItemFilename(configItem *kotsv1beta1.ConfigItem) string {
```

Comment:
this function wasn't used, remove it and its tests, we already have `configOptionFilename` for the template function and has similar tests.